src/rules/no-identical-title.ts

```diff
@@ -1,3 +1,4 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
```

Comment:
is this import used? other changes in file seems to be just whitespace now that it landed on master, so would be cool to avoid that diff?

Reply:
Yeah that's weird - I'll check this out.
Those whitespace changes were in my PR to master iirc, so not sure what's happened.

Feel free to drop changes to this file when rebasing :)